docs/_docs/reference/other-new-features/generalized-method-syntax.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: doc-page
+title: "Generalized Method Syntax"
+movedTo: https://docs.scala-lang.org/scala3/reference/other-new-features/generalized-method-syntax.html
+---
+
+The inclusion of using clauses is not the only way in which methods have been updated, type parameter clauses are now allowed in any number and at any position.
+
+## Syntax Changes
+
+### In Scala 2
+
+The old syntax only allowed zero or one type parameter clause, followed by any number of term clauses, optionnally followed by an implicit clause:
+
+```scala
+def foo[T, U](x: T)(y: U)(z: Int, s: String)(using Ord[Int])(a: Array[T])(implicit l: List[U])
```

Comment:
The section title is "In Scala 2" but there's a `using` clause here. That's not consistent.